tests/topology_test.go

```diff
@@ -0,0 +1,579 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package tests
+
+import (
+	"errors"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/redhat-cip/skydive/agent"
+	"github.com/redhat-cip/skydive/config"
+	"github.com/redhat-cip/skydive/logging"
+	"github.com/redhat-cip/skydive/topology/graph"
+)
+
+const conf = `
+[default]
+ws_pong_timeout = 1
+
+[agent]
+listen = 8081
+flowtable_expire = 5
+
+[cache]
+expire = 300
+cleanup = 30
+
+[sflow]
+listen = 5000
+
+[ovs]
+ovsdb = 6400
+
+[logging]
+default=DEBUG
```

Comment:
we should not print debug message during test ..... please remove the logging section or set it to INFO or even better ERROR